### ISSUE TITLE
Tooltip Alignment issue with watermark add-on settings

### ISSUE THREAD

**Issue by kjnanda** [rtMediaWP/rtMedia#1376]
```
Test Environment 
rtMedia Core: v4.5.6
BuddyPress: v4.1.0
rtMedia Photo watermark add-on: code from pre-release branch ( current version 1.2.5 ) 
```
1. Go to rtMedia Settings tab
2. Navigate to watermark settings page.
3. Go to font color and click on select color.
4. Observe that, when color picker is displayed, the `info` icon gets misaligned. Also, the `clear` button is also misaligned.

Refer below snapshot.
![ui-issue](https://user-images.githubusercontent.com/41000648/53093860-98d74500-353e-11e9-991d-b2d581229e08.png)

**Comment by kjnanda** [comment]
@surajkrsingh Please look into this issue

**Comment by surajkrsingh** [comment]
PR:-https://github.com/rtMediaWP/rtMedia/pull/1405/files

**Comment by kjnanda** [comment]
This is working as expected and tested with `pre-release` branch. ✅

**Comment by kjnanda** [comment]
This is working as expected and tested with `pre-release` branch. ✅

**Comment by kjnanda** [comment]
This is working as expected and tested with `dev` branch. ✅

**Comment by kjnanda** [comment]
This is working as expected and tested with `master` branch. ✅

**Comment by kjnanda** [comment]
Closing this issue as this has been fixed and release in - V 4.5.8
